README.md

```diff
@@ -30,35 +30,48 @@
-      - [LineageOS for microg](https://download.lineage.microg.org)
-      - [BlissRoms](https://blissroms.org)
-      - [PixelExperience](https://download.pixelexperience.org)
-    - TWRP Recovery:
-      - [TWRP recovery](https://twrp.me/Devices)
-    - Optional Addons:
-      - There are different packages of *Google Apps* available.
-        - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
-        - [NikGApps](https://nikgapps.com)
-      - [MicroG](https://microg.org)
-        - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
-      - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
-    - or you can just search the web or the [xda-developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
-3. Start the desktop app and follow the instructions.
+Linux is currently the best supported platform (tested with Ubuntu 20.04/22.04 LTS). Windows and macOS are also supported, but you might experience more issues.<br />
+So far there is no support for ARM-based systems.
+
+### 1. Download OpenAndroidInstaller
+
+Download the correct version for your OS [from the Releases](https://github.com/openandroidinstaller-dev/openandroidinstaller/releases).
+
+For Linux users, you can get the [official Flatpak version from Flathub](https://flathub.org/apps/org.openandroidinstaller.OpenAndroidInstaller). (You might need to change permissions to run the executable)
+
+For Windows users, you might need to [install the Universal USB Drivers](https://adb.clockworkmod.com) and other potentially drivers needed for your device.
+
+### 2. Download the custom ROM image & the TWRP recovery image (& optionally addons)
+
+Here are the official links for:
+
+  - some custom ROMs:
+    - [LineageOS](https://wiki.lineageos.org/devices)
+    - [/e/OS](https://doc.e.foundation/devices)
+    - [LineageOS for MicroG](https://download.lineage.microg.org)
+    - [BlissRoms](https://blissroms.org)
+    - [PixelExperience](https://download.pixelexperience.org)
+  - TWRP Recovery:
+    - [TWRP recovery](https://twrp.me/Devices)
+  - Optional Addons:
+    - There are different packages of *Google Apps* available:
+      - [MindTheGapps](https://wiki.lineageos.org/gapps#downloads)
+      - [NikGApps](https://nikgapps.com)
+    - [MicroG](https://microg.org)
+      - The recommended way to install MicroG is to use the zip file provided here: [https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases](https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases).
+    - [F-Droid App-Store](https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota).
+  - or you can just search the web or the [XDA Developers forum](https://forum.xda-developers.com) for an appropriate version for your device.
```

Comment:
The XDA Forums URL has changed to `https://xdaforums.com`

Reply:
Fixed